changelog.rst

```diff
@@ -1,3 +1,15 @@
+3.5.0 (2022/03/31)
```

Comment:
```suggestion
Upcoming
```
We don't know version number or date of next release yet. Just put "Upcoming" here for now, to be updated by the maintainer when releasing.